src/libraries/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoAMDesignator.cs

```diff
@@ -13,6 +14,205 @@ public void AMDesignator_GetInvariantInfo_ReturnsExpected()
+            yield return new object[] { new CultureInfo("pl-PL").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("pt-BR").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("pt-PT").DateTimeFormat, "da manhã" };
+            yield return new object[] { new CultureInfo("ro-RO").DateTimeFormat, "a.m." };
+            yield return new object[] { new CultureInfo("ru-RU").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("sk-SK").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("sl-SI").DateTimeFormat, "dop." };
+            yield return new object[] { new CultureInfo("sr-Cyrl-RS").DateTimeFormat, "AM" }; // пре подне
+            yield return new object[] { new CultureInfo("sr-Latn-RS").DateTimeFormat, "AM" }; // pre podne
+            yield return new object[] { new CultureInfo("sv-AX").DateTimeFormat, "fm" };
+            yield return new object[] { new CultureInfo("sv-SE").DateTimeFormat, "fm" };
+            yield return new object[] { new CultureInfo("sw-CD").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("sw-KE").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("sw-TZ").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("sw-UG").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("ta-IN").DateTimeFormat, "முற்பகல்" };
+            yield return new object[] { new CultureInfo("ta-LK").DateTimeFormat, "முற்பகல்" };
+            yield return new object[] { new CultureInfo("ta-MY").DateTimeFormat, "முற்பகல்" };
+            yield return new object[] { new CultureInfo("ta-SG").DateTimeFormat, "முற்பகல்" };
+            yield return new object[] { new CultureInfo("te-IN").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("th-TH").DateTimeFormat, "ก่อนเที่ยง" };
+            yield return new object[] { new CultureInfo("tr-CY").DateTimeFormat, "ÖÖ" };
+            yield return new object[] { new CultureInfo("tr-TR").DateTimeFormat, "ÖÖ" };
+            yield return new object[] { new CultureInfo("uk-UA").DateTimeFormat, "дп" };
+            yield return new object[] { new CultureInfo("vi-VN").DateTimeFormat, "SA" };
+            yield return new object[] { new CultureInfo("zh-CN").DateTimeFormat, "上午" };
+            yield return new object[] { new CultureInfo("zh-Hans-HK").DateTimeFormat, "上午" };
+            yield return new object[] { new CultureInfo("zh-SG").DateTimeFormat, "上午" };
+            yield return new object[] { new CultureInfo("zh-HK").DateTimeFormat, "上午" };
+            yield return new object[] { new CultureInfo("zh-TW").DateTimeFormat, "上午" };
+        }
+
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsHybridGlobalizationOnBrowser))]
```

Comment:
In the other case it would work as well but testing non-IsHybridGlobalizationOnBrowser case does not make sense because if we already test one or two locales for it, then we have a sufficient proof that whole ICU data got loaded correctly. Because of that, running all these tests would be just a waste of time for the CI. In the IsHybridGlobalizationOnBrowser scenario it's not the case, the fact that 5 or even 100 locales work does not mean that ~200 that we should support will work because of regexing and other operations on strings between dotnet API and Web API.

Reply:
Clarification to the previous statement: In some simpler APIs e.g. see [DateTimeFormatInfoAbbreviatedDayNames.cs](https://github.com/dotnet/runtime/pull/89255/files/b5f87001f2bd24cc43f13fea62b5165b2d0032f9#diff-027b3ff56ccde2e0278325fd48d7b5f0a659d150bf451feed56a8e77c8cbf445) we have less locales tested than the full set contains (one test case per region, not one test case per locale) because the transformations for these APIs are quite simple and we don't expect other locales from the region to differ.